core/storage/feedback/gae_models.py

```diff
@@ -204,16 +287,31 @@ def get_messages(cls, exploration_id, thread_id):
 
     @classmethod
     def get_most_recent_message(cls, exploration_id, thread_id):
+        """Returns most recent messages ordered by their last updated field.
```

Comment:
Returns the most recent messages from the given thread, ordered by their "last updated" field.